srcpkgs/pantheon-agent-polkit/files/io.elementary.desktop.agent-polkit.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=Authentication Dialog
+Exec=/usr/libexec/policykit-1-pantheon/io.elementary.desktop.agent-polkit
+Icon=dialog-password
+NoDisplay=true
+X-GNOME-Autostart-Phase=Initialization
+X-GNOME-Autostart-Notify=true
+X-GNOME-AutoRestart=true
```

Comment:
Are the `X-GNOME-AutoRestart*` lines suitable for upstream? Getting them added in the official release would allow you to phase out the custom file in a later update. Until then, or if the Pantheon team won't accept GNOME-specific content in the file (I don't know what Elementary does for X session management), using a custom version of the file seems sensible.

Reply:
You mean , they should be remove?

Reply:
No. I think you are doing the right thing here. But you should also open an issue upstream to see if these lines should be included with the official release of `pantheon-agent-polkit`. If they are eventually included upstream, then you will be able to remove the custom file.

Reply:
Sure.